service/resource/persistentvolume/create.go

```diff
@@ -0,0 +1,17 @@
+package persistentvolume
+
+import (
+	"context"
+
+	"github.com/giantswarm/operatorkit/framework"
+)
+
+// NewCreatePatch is not used
```

Comment:
Nitpick but please make the GoDoc comments full sentences with a full stop. It makes me and @xh3b4sd sad when we don't do this. 😄